guacamole/src/main/frontend/src/translations/fr.json

```diff
@@ -51,8 +64,12 @@
     "CLIENT" : {
 
         "ACTION_ACKNOWLEDGE"               : "@:APP.ACTION_ACKNOWLEDGE",
-        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "Vider",
+        "ACTION_CANCEL"                    : "@:APP.ACTION_CANCEL",
+        "ACTION_CLEAR_CLIENT_MESSAGES"     : "@:APP.ACTION_CLEAR",
+        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "@:APP.ACTION_CLEAR",
+        "ACTION_CONTINUE"                  : "@:APP.ACTION_CONTINUE",
```

Comment:
I can't remember exactly how these work, but I didn't think it was necessary to include all of the references to other sections/translations - I thought that, this, too, would be inherited from the English translation. But, I can't remember if that results in things being translated in English or the intended language...

Reply:
I've seen them in various places, so I preferred to add them for consistency with the rest of the file.  
No problem removing them if you prefer.

Reply:
We'll leave them for now - we can always clean them up later if they aren't required.